core/src/main/java/discord4j/core/object/ExtendedInvite.java

```diff
@@ -45,12 +45,13 @@ public ExtendedInvite(final ServiceMediator serviceMediator, final ExtendedInvit
     }
 
     /**
-     * Gets the ID of the user who created the invite.
+     * Gets the ID of the user who created the invite, if present.
      *
-     * @return The ID of the user who created the invite.
+     * @return The ID of the user who created the invite, if present.
      */
-    public Snowflake getInviterId() {
-        return Snowflake.of(getData().getInviterId());
+    public Optional<Snowflake> getInviterId() {
```

Comment:
We don't break compatibility in x.y.z releases so this is not allowed for the 3.0.x branch. You could move this to a separate method, but deprecate it immediately because in 3.1 this is already fixed

Reply:
Maybe create a getOptionalInviterId with the Optional? and keep getInviterId with Deprecated?

Reply:
Ok to the new method, but deprecate it instead so we can remove it when merging this to master.

`getInviterId` should be nullable- I'm not sure if this should also be tagged as Deprecated notifying about return type changes for 3.1 🤔 so I would leave it like that

Reply:
The new method have the "Experimental" annotation (for draft methods) for the remove/rename and the getInviterId well its not Nullable, or get the Snowflake or throw a NullPointerException, or need change the return for get the null and not throw the exception.